sunpy/coordinates/ephemeris.py

```diff
@@ -34,7 +34,8 @@
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
-def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include_velocity=False):
+def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include_velocity=False,
+                                     quiet=False):
```

Comment:
Would `verbose` be a more standard/canonical argument name to use here I'm +/- 0 either way, just thought I'd throw the suggestion out there.

Reply:
<shrug>  I guess I think of "verbose output" as more than the default level of output, so it doesn't make sense to me for `verbose=True` to be a default setting.